lib/test.js

```diff
@@ -52,11 +52,22 @@ function Test (name_, opts_, cb_) {
     this.pendingCount = 0;
     this._skip = args.opts.skip || false;
     this._timeout = args.opts.timeout;
-    this._objectPrintDepth = args.opts.objectPrintDepth || 5;
     this._plan = undefined;
     this._cb = args.cb;
     this._progeny = [];
     this._ok = true;
+    var depthEnvVar = 'NODE_TAPE_OBJECT_PRINT_DEPTH';
+    if (args.opts.objectPrintDepth) {
+        this._objectPrintDepth = args.opts.objectPrintDepth
+    } else if (process.env[depthEnvVar]) {
+        if (process.env[depthEnvVar].toLowerCase() === 'infinity') {
+            this._objectPrintDepth = Infinity;
+        } else {
+            this._objectPrintDepth = parseInt(process.env[depthEnvVar], 10);
```

Comment:
hm, i'm not actually sure about the parseInt here. tape doesn't seem to do any typechecking on the `objectPrintDepth` option - ie, you could pass a non-number or a non-integer and it'd just coerce to a number later. Maybe we should do the same here?

Reply:
ok